newlib/libc/sys/arc/syscalls.c

```diff
@@ -35,6 +37,10 @@ _syscall2 (int,gettimeofday, struct timeval *,tv, void *,tz)
 
 _syscall1 (void,exit, int,ret)
 
+_syscall1 (_CLOCK_T_,times, struct tms *,buf)
```

Comment:
I've supposed that there is a naming convention exist for this file. E.g. other `_syscallX` macros are used the same way here. I can reformat the whole file if it's necessary.